AtomicU64 unresolved symbol

`AtomicU64` does not exist in some systems.

I was trying to compile sccache in a powerpc (32-bit) vm with ArchPOWER.
It failed because of this library:

```
...
   Compiling bb8 v0.8.6
error[E0432]: unresolved import `std::sync::atomic::AtomicU64`
 --> /home/user/.cargo/registry/src/index.crates.io-9850fa1df3136423/bb8-0.8.6/src/internals.rs:3:25
  |
3 | use std::sync::atomic::{AtomicU64, Ordering};
  |                         ^^^^^^^^^
  |                         |
  |                         no `AtomicU64` in `sync::atomic`
  |                         help: a similar name exists in the module: `AtomicU32`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `bb8` (lib) due to 1 previous error
error: failed to compile `sccache v0.9.0`, intermediate artifacts can be found at `/tmp/cargo-installkWejfU`.
...
```

--- a/bb8/Cargo.toml
+++ b/bb8/Cargo.toml
@@ -14,6 +14,9 @@ futures-util = { version = "0.3.2", default-features = false, features = ["alloc
 parking_lot = { version = "0.12", optional = true }
 tokio = { version = "1.0", features = ["rt", "sync", "time"] }
 
+[target.'cfg(not(target_has_atomic = "u64"))'.dependencies]
+portable-atomic = "1"
+
 [dev-dependencies]
 tokio = { version = "1.0", features = ["macros"] }
 

--- a/bb8/src/internals.rs
+++ b/bb8/src/internals.rs
@@ -1,6 +1,10 @@
+#[cfg(not(target_has_atomic = "64"))]
+use portable_atomic::AtomicU64;
 use std::cmp::min;
 use std::collections::VecDeque;
-use std::sync::atomic::{AtomicU64, Ordering};
+#[cfg(target_has_atomic = "64")]
+use std::sync::atomic::AtomicU64;
+use std::sync::atomic::Ordering;
 use std::sync::Arc;
 use std::time::{Duration, Instant};
 

--- a/bb8/src/internals.rs
+++ b/bb8/src/internals.rs
@@ -1,5 +1,3 @@
-#[cfg(not(target_has_atomic = "64"))]
-use portable_atomic::AtomicU64;
 use std::cmp::min;
 use std::collections::VecDeque;
 #[cfg(target_has_atomic = "64")]
@@ -8,6 +6,8 @@ use std::sync::atomic::Ordering;
 use std::sync::Arc;
 use std::time::{Duration, Instant};
 
+#[cfg(not(target_has_atomic = "64"))]
+use portable_atomic::AtomicU64;
 use tokio::sync::Notify;
 
 use crate::api::{Builder, ManageConnection, QueueStrategy, State, Statistics};

